libunsf.c

```diff
@@ -3324,7 +3339,7 @@ UNSF_SYMBOL void unsf_convert_sf_to_gus(UnSF_Options *options) {
                                     if ((sf_num_instrument_indexes * 4 != subchunk.size) ||
                                         (sf_instrument_indexes)) BAD_SF();
 
-                                    sf_instrument_indexes = malloc(sizeof(sfInstBag) * sf_num_instrument_indexes);
+                                    sf_instrument_indexes = (sfInstBag *) malloc(sizeof(sfInstBag) * sf_num_instrument_indexes);
```

Comment:
On 9/6/16, Bret Curtis <notifications@github.com> wrote:
> No, WildMIDI... since unsf will be used in WildMIDI eventually.

Possibly, but it is still a standalone library too.

About libunsf to be used in wildmidi:  it won't be a straightforward
task, I hope it won't add any additional bugs, But that's off-topic here.